include/cell.h

```diff
@@ -249,8 +274,19 @@ class Cell {
 
   //! Shape function
   std::shared_ptr<const Element<Tdim>> element_{nullptr};
+
+  //! Velocity constraints
+  //! first-> face_id, second->pair of direction [0/1/2] and velocity value
+  // std::vector<std::tuple<unsigned, unsigned, double>> velocity_constraints_;
+  std::map<unsigned, std::vector<std::tuple<unsigned, double>>>
```

Comment:
You couldn't use a pair?

Reply:
Right, will use pair. I was in a rush just now, and stick with what I had before :(